lib/routes.go

```diff
@@ -0,0 +1,71 @@
+package router
+
+import (
+	"encoding/json"
+)
+
+type CsRoute struct {
+	IncomingPath *string
+	RouteType    *string
+	BackendID    *string
+	RedirectTo   *string
+	SegmentsMode *string
+	SchemaName   *string
+	Details      *string
+}
+
+func (route *CsRoute) backend() *string {
+	if route.SchemaName != nil && *route.SchemaName == "gone" && !route.gone() {
+		if route.BackendID != nil {
+			return route.BackendID
+		} else {
+			defaultBackend := "government-frontend"
+			return &defaultBackend
```

Comment:
`defaultBackend` is only in scope for `else` and not used elsewhere. This can just be `return "government-frontend"`

Reply:
Sorry, I realised it needs to return a string pointer. You need to create the variable separately and then return the reference.